docs/releases.rst

```diff
@@ -331,7 +331,7 @@ Features
 
 - Improved error messaging for exceptions from malformed JSON-RPC responses. (`#3053 <https://github.com/ethereum/web3.py/issues/3053>`__)
 - Enable filtering by non-indexed arguments for contract event ``get_logs()``. (`#3078 <https://github.com/ethereum/web3.py/issues/3078>`__)
-- Add ``eth_maxPriorityFeePerGas`` to ``exception_retry_middleware`` whitelist (`#3090 <https://github.com/ethereum/web3.py/issues/3090>`__)
+- Add ``eth_maxPriorityFeePerGas`` to ``exception_retry_middleware`` allowlist (`#3090 <https://github.com/ethereum/web3.py/issues/3090>`__)
```

Comment:
feel free to take or leave, but I think since this was called the `whitelist` at this point in time, it makes sense to leave it?

Reply:
Agree with you here, I am looking for ways to ignore this case but still check `releases.rst`. There are inline ignores for code but not docs unfortunately.

I think since all of these come from commit messages and those are unchangeable we wouldn't want to change these here. I think I'll just ignore this file entirely.